### ISSUE TITLE
[BUG]: invalid dates when table.nulls = true and table.timestamps = true

### ISSUE THREAD

**Issue by demsey2** [sensedeep/dynamodb-onetable#92]
in Model.js we have transformWriteAttribute method

```
transformWriteAttribute(op, field, value) {
        let type = field.type;
        if (field.nulls === true) {
            ;
        }
        else if (op == 'find' && value != null && typeof value == 'object') {
            //  Find used {begins} and other operators
            value = this.transformNestedWriteFields(field, value);
        }
        else if (type == Date) {
            value = this.transformWriteDate(value);
        }
        else if (type == Number) {
            let num = Number(value);
            if (isNaN(num)) {
                throw new Error(`Invalid value "${value}" provided for field "${field.name}"`);
            }
            value = num;
        }
....
```

if field.nulls === true then we have that semicolon. Why is that?

I have spotted that when I setup my tables with nulls = true and timestamps = true then created and updated at are set to new Date() however they are not transformed because we end up in that first condition field.nulls === true

**Comment by mobsense** [comment]
Should be && value == null I think:

```
        if (value == null && field.nulls === true) {

```

**Comment by mobsense** [comment]
Fixed now in 1.7.1
